Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -949,6 +949,73 @@ public struct HTTPClientError: Error, Equatable, CustomStringConvertible {
         return "HTTPClientError.\(String(describing: self.code))"
     }
 
+    public var shortDescription: String {
```

Comment:
Would you mind adding a comment that describes, when users should use `shortDescription` and what they need to take into consideration when adding another case?

Reply:
Added now. Happy to adjust the comments of course!